concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1037,6 +978,104 @@ jobs:
+                  mkdir certs
+                  echo "Extracting certs"
+                  tar -xvzf cf-certs/cf-certs.tar.gz -C certs
+
+                  for cert_entry in apps_domain system_domain; do
+                    rm -vf "certs/${cert_entry}.key" "certs/${cert_entry}_old.key" \
+                           "certs/${cert_entry}.csr" "certs/${cert_entry}_old.csr" \
+                           "certs/${cert_entry}.crt" "certs/${cert_entry}_old.crt"
+                  done
+
+                  echo "Creating updated cert tarball"
+                  cd certs
+                  tar -cvzf ../generated-cf-certs/cf-certs.tar.gz .
+          on_success:
+            put: cf-certs
+            params:
+              file: generated-cf-certs/cf-certs.tar.gz
+        - try:
+            get: cf-certs-tfstate
+            on_success:
+              task: undo-upload-certs-to-aws
+              config:
+                platform: linux
+                image_resource:
+                  type: docker-image
+                  source:
+                    repository: governmentpaas/terraform
+                    tag: 9cad30b5d5889a0b72173f39701d1620e24df82c
+                inputs:
+                  - name: paas-cf
+                  - name: cf-certs-tfstate
+                outputs:
+                  - name: updated-tfstate
+                params:
+                  TF_VAR_region: ((aws_region))
+                run:
+                  path: sh
+                  args:
+                    - -e
+                    - -c
+                    - |
+                      cp cf-certs-tfstate/cf-certs.tfstate updated-tfstate/cf-certs.tfstate
+                      terraform init paas-cf/terraform/cf-certs
+                      terraform destroy \
+                        -force \
+                        -var-file="paas-cf/terraform/((aws_account)).tfvars" \
+                        -state=updated-tfstate/cf-certs.tfstate \
+                        paas-cf/terraform/cf-certs
+              on_success:
+                do:
+                - put: cf-certs-tfstate
+                  params:
+                    file: updated-tfstate/cf-certs.tfstate
+
+                - task: delete-old-cf-certs-tfstate
+                  config:
+                    platform: linux
+                    inputs:
+                      - name: paas-cf
+                    image_resource:
+                      type: docker-image
+                      source:
+                        repository: governmentpaas/awscli
+                        tag: b2495d6ed07f680125d19aa7d1701da7efabb289
+                    params:
+                      AWS_DEFAULT_REGION: ((aws_region))
+                      STATE_BUCKET: ((state_bucket))
+                    run:
+                      path: sh
+                      args:
+                        - -e
+                        - -c
+                        - |
+                          aws s3 rm "s3://${STATE_BUCKET}/cf-certs.tfstate"
```

Comment:
Likewise, removing this file is likely to cause future runs of the pipeline to stall.